website/docs/patterns/filter-patterns/partials/specifications/specifications.md

```diff
@@ -0,0 +1,139 @@
+| Segmented Input | Allows for searching and filtering within data set on string and text values |
+| Segmented Button | Allows functions to be performed on the data set, e.g., refreshing the dataset. |
+| Segmented Dropdown | Displays the filter parameter on the toggle button and numerous filterable values in a dropdown list. |
+| [List](/components/dropdown#list) | Coupled with the [Dropdown](/components/dropdown), the List displays filterable values that correspond with the parameter in the Dropdown Toggle |
+
+### Spacing
+
+In the case of multiple Segmented Groups within a filter bar, use a `16px` gap between each group.
+
+![Multiple groups in a filter bar](/assets/patterns/filter-patterns/filter-bar-anatomy-02.png)
+
+Sometimes a set of global filter functions is necessary to refresh the data set or surface an action to reveal additional overflow filters in a Flyout. In this case the global functions should be aligned to the end of the container with space between the primary filters and the global functions.
+
+![Functions within a filter bar](/assets/patterns/filter-patterns/filter-bar-anatomy-03.png)
+
+## Filter sidebar
+
+With more real estate in a filter sidebar, the list that was responsible for displaying the filter values in the filter bar can be expanded to display all applicable values at once in a group.
+
+![Sidebar anatomy](/assets/patterns/filter-patterns/sidebar-anatomy.png =350x*)
+
+| Element | Usage |
+|---------|-------|
+| Sidebar container | Wraps the filters or multiple groups in a vertical orientation. |
+| Divider | (Optional, but recommended) Increases visual differentiation between each group. |
+| [Checkbox Group](/components/form/checkbox) | Displays the values in a checkbox group, supporting the selection of multiple items. |
+| [Radio Group](/components/form/radio) | Displays the values in a radio group, supporting the selection of a single value. |
+
+### Spacing
+
+Spacing within a filter sidebar extends the guidelines for spacing within a [form pattern](/patterns/form-patterns).
+
+- If a divider separates the groups use a `24px` vertical gap on the top and bottom of the divider.
+- If no divider is used, groups should receive a `32px` gap between them.
+
+**Ex 3.0: Spacing in a sidebar that uses dividers**
+
+![Sidebar spacing](/assets/patterns/filter-patterns/sidebar-spacing-divider.png =350x*)
+
+**Ex 3.1: Spacing in a sidebar that doesn't use dividers**
+
+![Sidebar spacing without a divider](/assets/patterns/filter-patterns/sidebar-spacing-no-divider.png =350x*)
+
+### Filters within a Flyout
+
+If its necessary for filters to overflow into a [Flyout](/components/flyout), use the same organizational method and spacing guidelines as the filter sidebar.
+
+![Filters within a Flyout](/assets/patterns/filter-patterns/flyout-filter-spacing.png)
```

Comment:
While it has a similar structure, you don't really refer to this as a subset of the filter sidebar, so it feels weird to nest the content under a Sidebar heading. I recommend bumping it up a level. 

```suggestion
## Overflow filters

If it's necessary for filters to overflow into a [Flyout](/components/flyout), use the same organizational method and spacing guidelines as the filter sidebar.

![Filters within a Flyout](/assets/patterns/filter-patterns/flyout-filter-spacing.png)
```